examples/defmo_example.py

```diff
@@ -0,0 +1,149 @@
+import cv2
```

Comment:
@rozumden skip this example for now from this PR

Reply:
I agree

Reply:
Done.